item.rb

```diff
@@ -0,0 +1,42 @@
+class Item
```

Comment:
Kindly add the tests for the `Item` class. Tests should cover all the methods in this class.